lib/api/funnel.js

```diff
@@ -368,9 +369,29 @@ class Funnel {
    * @returns {Promise<Request>}
    */
   async checkRights (request) {
+   
+    let skipTokenVerification = false;
+    // When the Support of Cookie as Jwt is enabled we check if a jwt cookie is present
+    // When a request is made with CookieOnly set to true
+    // We try to use the jwt cookie if present as jwt token
+    // otherwise check for jwt as input
+    if (global.kuzzle.config.security.supportCookieAuthentication
```

Comment:
Nitpicking: coding standard
```suggestion
    if ( global.kuzzle.config.security.supportCookieAuthentication
```